tests/resources/trailer-registration.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is it possible to add some tests for when something goes wrong in the handler?
We added a catch block with this
```js
console.error(`Error on fetching vinOrChassisWithMake ${vin+make}`, e);
```

I need to catch up with you to understand the testing a bit more if it is possible please.